### PR TITLE
Support set storage group in standalone mpp mode

### PR DESCRIPTION
**Pull request by HTHou**
## Description

1. move LocalConfigNode to localconfignode package
2. implement some interfaces of Standalone schema fetcher and Standalone schema fetcher 
3. support set storage group in standalone mpp mode.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/service/DataNode.java [hunk 1]
```diff
@@ -170,6 +170,7 @@ public void joinCluster() throws StartupException {
   public void active() throws StartupException {
     // set the mpp mode to true
     IoTDBDescriptor.getInstance().getConfig().setMppMode(true);
+    IoTDB.getInstance().setClusterMode();
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/service/DataNode.java:173]:
```suggestion
```

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/config/SetStorageGroupTask.java [hunk 2]
```diff
@@ -45,31 +48,39 @@ public SetStorageGroupTask(SetStorageGroupStatement setStorageGroupStatement) {
...
-      if (TSStatusCode.SUCCESS_STATUS.getStatusCode() != tsStatus.getCode()) {
-        LOGGER.error(
-            "Failed to execute set storage group {} in config node, status is {}.",
-            setStorageGroupStatement.getStorageGroupPath(),
-            tsStatus);
-        future.setException(new StatementExecutionException(tsStatus));
-      } else {
-        future.set(new ConfigTaskResult(TSStatusCode.SUCCESS_STATUS));
+    if (IoTDB.getInstance().isMppClusterMode()) {
+      // Construct request using statement
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/mpp/execution/config/SetStorageGroupTask.java:52]: Is this ```if-else``` a temporary implementation? I think this judgement should be integrated in a high level framework?

**Review by MarcosZyk — COMMENTED** [review]
Excellent work！Only one question. Please take a look.

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/config/SetStorageGroupTask.java (hunk 2 above)
Comment (HTHou) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/config/SetStorageGroupTask.java]: Yes, it is a temporary implementation. We need to fix it in the future. Let me add a TODO mark.

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**